test/stress/generate-report/generate_report.py

```
@@ -0,0 +1,860 @@
+            "podScheduled_p90": row[7],
+            "podRunning_p50": row[8],
+            "podRunning_p90": row[9],
+            "sandboxReady_p50": row[10],
+            "sandboxReady_p90": row[11],
+            "sandboxReady_p99": row[12]
+        })
+
+    # 5. Render Templates using Jinja2
+    output_dir.mkdir(parents=True, exist_ok=True)
+    template_dir = Path(__file__).parent / "templates"
+    env = Environment(loader=FileSystemLoader(template_dir))
```

Ruff (0.15.21)

[error] By default, jinja2 sets `autoescape` to `False`. Consider using `autoescape=True` or the `select_autoescape` function to mitigate XSS vulnerabilities.

(S701)